src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -267,6 +274,12 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_deployment_operations_at_subscription_scope')
         g.custom_show_command('show', 'get_deployment_operations_at_subscription_scope', client_factory=cf_deployment_operations)
 
+    with self.command_group('deployment-scripts', resource_deploymentscripts_sdk, resource_type=ResourceType.MGMT_RESOURCE_DEPLOYMENTSCRIPTS, is_preview=True) as g:
+        g.custom_command('list', 'list_deployment_scripts')
+        g.custom_show_command('show', 'get_deployment_script')
+        g.custom_command('logs', 'get_deployment_script_logs')
```

Comment:
Same as above.
> It is recommended to use verbs to indicate the operation.
If there are other log operations in the future, I suggest changing it to `deployment-scripts log show`. 
If there is only show operation, I suggest changing it to `deployment-scripts show-log`